.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+	"presets": [
+		"@babel/preset-env"
+		],
+	"plugins": [
+		"@babel/plugin-transform-runtime"
+	]
+}
```

Comment:
Add a new line here and let your indentation be two spaces

Reply:
I have done that. Please check now